android-components/automation/taskcluster/androidTest/parse-ui-test.py

```diff
@@ -57,11 +57,19 @@ def main():
     print("| matrix | result | logs | details \n")
     print("| --- | --- | --- | --- |\n")
     for matrix, matrix_result in matrix_ids.items():
-        print("| {matrixId} | {outcome} | [logs]({webLink}) | {axes[0][details]}\n".format(**matrix_result))
+        print(
+            "| {matrixId} | {outcome} | [logs]({webLink}) | {axes[0][details]}\n".format(
+                **matrix_result
+            )
+        )
     print("---\n")
     print("# References & Documentation\n")
-    print("* [Automated UI Testing Documentation](https://github.com/mozilla-mobile/shared-docs/blob/main/android/ui-testing.md)\n")
-    print("* Mobile Test Engineering on [Mana](https://mana.mozilla.org/wiki/display/MTE/Mobile+Test+Engineering) | [Slack](https://mozilla.slack.com/archives/C02KDDS9QM9) | [Alerts](https://mozilla.slack.com/archives/C0134KJ4JHL)\n")
+    print(
+        "* [Automated UI Testing Documentation](https://github.com/mozilla-mobile/shared-docs/blob/main/android/ui-testing.md)\n"
+    )
+    print(
+        "* Mobile Test Engineering on [Mana](https://mana.mozilla.org/wiki/display/MTE/Mobile+Test+Engineering) | [Slack](https://mozilla.slack.com/archives/C02KDDS9QM9) | [Alerts](https://mozilla.slack.com/archives/C0134KJ4JHL)\n"
```

Comment:
looks like the wiki link is now https://mozilla-hub.atlassian.net/wiki/spaces/MTE/overview, we might want to update the various in-tree links in a followup.

Reply:
Filed bug 1843157.